pkg/manager/internal/controllers/project_controller.go

```diff
@@ -108,12 +113,40 @@ func (r *ProjectReconciler) reconcileMembers(ctx context.Context, project *stora
 	return nil
 }
 
+// Project Namespace name is organization_name-separator-project_name
+func generateProjectNamespaceName(organizatioName, projectName string) string {
+	return fmt.Sprintf("%s%s%s", organizatioName, projectNamespaceNameSeparator, projectName)
+}
+
+// Project Namespace name is organization_name-separator-project_name
+func deconstructProjectNamespaceName(projectNamespaceName string) (string, string, error) {
```

Comment:
don't use parsing namespace name for anything structured; if needed write it in the metadata annotation; parsing is tricky.